third_party/externals.bzl

```diff
@@ -310,3 +310,11 @@ def register_sorbet_dependencies():
         sha256 = "47b61d25dd52bdaa1d571dab6705d076f05ba3d7a1bbbfed36145f8281c0403f",
         strip_prefix = "rules_foreign_cc-d74623f0ad47f4e375de81baa454eb106715a416",
     )
+
+    http_archive(
+        name = "rbs_parser",
+        url = "https://github.com/shopify/rbs/archive/6c66d8cdcb3ff70cb401fb6dfba43dd3e45b56aa.zip",
```

Comment:
We're currently working out of a branch that contains our refactor of the RBS parser. We're upstreaming the changes as we speak but it will take some time before the ~200 refactor commits reach `master`.